### PR TITLE
Ux improvements

### PR DESCRIPTION
**Pull request by OnikNoor97**

### CODE REVIEW

File: src/Script/admin.js [hunk 1]
```diff
@@ -194,4 +209,37 @@ function updateUserLevel(userIdSelected)
...
+    $("#admin-modal-body").html("").append(adminEditDiv);
+
+    $('#editUserSelecter').on('change',(event) => {
+
+
+        (event.target.value == 1) ? $('#admin-modal-footer').children().remove()
+        : (event.target.value == 2) ? $("#admin-modal-footer").html("").append($("<button>", {class : "btn btn-danger", type : "text", id : "updateUserBtn1", onclick : `updateUserLevelPrompt(${userIdSelected})`}).html("Update User"))
+        : (event.target.value == 3) ? $("#admin-modal-footer").html("").append($("<button>", {class : "btn btn-danger", type : "text", id : "updateUserBtn2", onclick : `editPasswordPrompt(${userIdSelected})`}).html("Edit Password (wip)"))
+        : (event.target.value == 4) ? $("#admin-modal-footer").html("").append($("<button>", {class : "btn btn-danger", type : "text", id : "updateUserBtn3", onclick : `passwordResetPrompt(${userIdSelected})`}).html("Reset Password"))
+        : null;
```
Comment (OnikNoor97) [inline src/Script/admin.js:243]: Use If statements, use Ternary Operators to set values, this is bad practice
Also get rid of, 'change password' Admins should not have the power to change the password (WIP will never be used anymore, lmaoo is on beta)

File: src/Css/projectPage.css [hunk 2]
```diff
@@ -22,14 +22,13 @@
...
-    color: #000;
-	padding: 10px;
+	color: #000;
 	border: none;
 	background: none;
 }
 
 #sidebar ul li {
-    padding: 10px;
+	padding: 14px;
```
Comment (OnikNoor97) [inline src/Css/projectPage.css:31]: Fix indent

Comment (LewisWelch94) [reply src/Css/projectPage.css:31]: fixed

File: src/Script/admin.js (hunk 1 above)
Comment (LewisWelch94) [reply src/Script/admin.js]: fixed

File: src/Script/admin.js [hunk 3]
```diff
@@ -196,4 +211,34 @@ function updateUserLevel(userIdSelected)
...
+    $(adminSelecter).append(adminSelectoption1, adminSelectoption2, adminSelectoption3);
+
+    $("#admin-modal-body").html("").append(adminEditDiv);
+
+    $('#editUserSelecter').on('change',(event) => {
+
+        if (event.target.value == 1) $('#admin-modal-footer').children().remove();
+         else if (event.target.value == 2) $("#admin-modal-footer").html("").append($("<button>", {class : "btn btn-danger", type : "text", id : "updateUserBtn1", onclick : `updateUserLevelPrompt(${userIdSelected})`}).html("Update User"));
+            else if (event.target.value == 3) $("#admin-modal-footer").html("").append($("<button>", {class : "btn btn-danger", type : "text", id : "updateUserBtn3", onclick : `passwordResetPrompt(${userIdSelected})`}).html("Reset Password"));
+                else null;
```
Comment (OnikNoor97) [inline src/Script/admin.js:242]: why is null here? use property if statements with curly brackets

Comment (LewisWelch94) [reply src/Script/admin.js]: fixed

**Review by OnikNoor97 — COMMENTED** [review]
Approved

**Pull request merged**